### PR TITLE
Fix: Ethnicity field autopopulates to Unknown

### PR DESCRIPTION
**Pull request by GPaoloni**
Jira: https://bugs.benetech.org/browse/CHI-862
Primary reviewer: @murilovmachado 

This PR:
- Changes `getAnswerOrUnknown` function to return `null` if a requested `key` is not part of the `answers` object (bot's memory). By request, `age` and `gender` are excepted: `Unknown` is returned even if there's no answer.
- Before dispatching `prepopulateFormAction`, fill the `values` argument object only with the keys of pre-survey that actually contains a value (a [truthy value](https://developer.mozilla.org/en-US/docs/Glossary/Truthy)). The edge case for `"0"` works fine because is not an integer, but a non-empty string.

What to test here:
- Pre survey continues to work as expected.
- Pre survey does not fills ethnicity for ZM, as it's not part of the questions (thus `answers['ethnicity']` is undefined).
- Pre survey does fills ethnicity for BR. To easily test this, you can modify the task attributes while it is still pending to include a value for `answers['ethnicity']`, mimicking the behavior if this question would have been part of pre survey. When accepting the task, this value should be taken into consideration.

Note: it would be great at some point, have a little time to include unit tests for this piece of code.

### CODE REVIEW

File: plugin-hrm-form/src/utils/prepopulateForm.ts [hunk 1]
```diff
@@ -39,13 +39,20 @@ const getAnswerOrUnknown = (
   definition: FormDefinition,
   mapperFunction: MapperFunction = mapGenericOption,
 ) => {
+  // This keys must be set with 'Unknown' value even if there's no answer
+  const isRequiredKey = key === 'age' || key === 'gender';
```
Comment (murilovmachado) [inline plugin-hrm-form/src/utils/prepopulateForm.ts:43]: These hardcoded assumptions make me think that would be very good to have autopilot definitions (in the future). For now, this is fine.

File: plugin-hrm-form/src/utils/prepopulateForm.ts [hunk 2]
```diff
@@ -74,11 +81,11 @@ export const prepopulateForm = (task: ITask) => {
...
-      firstName,
-      gender,
-      age,
-      ethnicity,
-      language: capitalize(language),
+      ...(firstName && { firstName }),
+      ...(gender && { gender }),
+      ...(age && { age }),
+      ...(ethnicity && { ethnicity }),
+      ...(language && { language: capitalize(language) }),
```
Comment (murilovmachado) [inline plugin-hrm-form/src/utils/prepopulateForm.ts:88]: Nice, I think this is better because we won't have an object with `undefined` values (as before).

**Review by murilovmachado — APPROVED** [review]
Looks good to me
